src/dotnet/APIView/ClientSPA/src/app/_components/review-page/review-page.component.ts

```diff
@@ -361,10 +363,32 @@ export class ReviewPageComponent implements OnInit {
       .pipe(takeUntil(this.destroy$)).subscribe({
         next: (comments: CommentItemModel[]) => {
           this.comments = comments;
+          CommentRelationHelper.calculateRelatedComments(this.comments);
+          this.processEmbeddedComments();
         }
       });
   }
 
+  private processEmbeddedComments() {
+    if (!this.codePanelData || !this.comments) return;
+    
+    Object.values(this.codePanelData.nodeMetaData).forEach(nodeData => {
+      if (nodeData.commentThread) {
+        Object.values(nodeData.commentThread).forEach(commentThreadRow => {
+          if (commentThreadRow.comments) {
+            commentThreadRow.comments.forEach(embeddedComment => {
+              const globalComment = this.comments.find(c => c.id === embeddedComment.id);
```

Comment:
What is a global comment or an embedded comment?

Reply:
So if they are only visible in conversations but do match a correlation id, shouldn't they still be counted as related?